tests/final_chain_test.cpp

```diff
@@ -48,7 +48,8 @@ struct FinalChainTest : WithDataDir {
     }
     DagBlock dag_blk({}, {}, {}, trx_hashes, {}, secret_t::random());
     db->saveDagBlock(dag_blk);
-    auto pbft_block = std::make_shared<PbftBlock>(blk_hash_t(), blk_hash_t(), blk_hash_t(), 1, addr_t::random(),
+    std::vector<vote_hash_t> reward_votes;
+    auto pbft_block = std::make_shared<PbftBlock>(blk_hash_t(), blk_hash_t(), blk_hash_t(), 1, reward_votes,
```

Comment:
you could do it same way as you did for rest of the test with just `{}`

Reply:
"{}" doesn't work for make_shared

/Users/qigao/Taraxa_debug/taraxa-node/tests/final_chain_test.cpp:53:9: error: no matching function for call to 'make_shared'
        std::make_shared<PbftBlock>(blk_hash_t(), blk_hash_t(), blk_hash_t(), 1, {}, dev::KeyPair::create().secret());
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/opt/llvm@12/bin/../include/c++/v1/memory:3387:17: note: candidate template ignored: substitution failure [with _Tp = taraxa::PbftBlock]: deduced incomplete pack <dev::FixedHash<32>, dev::FixedHash<32>, dev::FixedHash<32>, int, (no value), const dev::SecureFixedHash<32> &> for template parameter '_Args'
shared_ptr<_Tp> make_shared(_Args&& ...__args)